src/zjs_event.c

```diff
@@ -40,50 +40,84 @@ void post_event(void* h, jerry_value_t* ret_val)
     }
 }
 
+static uint32_t get_num_events(jerry_value_t emitter)
+{
+    jerry_value_t val = zjs_get_property(emitter, "numEvents");
+    if (!jerry_value_is_number(val)) {
+        ERR_PRINT("emitter had no numEvents property\n");
+        return 0;
+    }
+    uint32_t num = jerry_get_number_value(val);
+    jerry_release_value(val);
+    return num;
+}
+
+static uint32_t get_max_event_listeners(jerry_value_t emitter)
+{
+    jerry_value_t val = zjs_get_property(emitter, "maxListeners");
+    if (!jerry_value_is_number(val)) {
+        ERR_PRINT("emitter had no maxListeners property\n");
+        return 0;
+    }
+    uint32_t num = jerry_get_number_value(val);
+    jerry_release_value(val);
+    return num;
+}
+
+static int32_t get_callback_id(jerry_value_t event_obj)
+{
+    int32_t callback_id = -1;
+    jerry_value_t id_prop = zjs_get_property(event_obj, "callback_id");
```

Comment:
This object is a property of a hidden object so there should be no way for them to access it.